FirebasePerformance/Sources/FPRClient.m

```diff
@@ -113,6 +113,19 @@ - (instancetype)init {
     _configuration = [FPRConfigurations sharedInstance];
     _projectID = [FIROptions defaultOptions].projectID;
     _bundleID = [FIROptions defaultOptions].bundleID;
+
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+      if (self.installations == nil) {
+        // Delayed initialization of installations because FIRApp needs to be configured first.
+        self.installations = [FIRInstallations installations];
```

Comment:
Curious if we moved this to initializer now. IIRC this was intentionally initialized later to not eat up the startup time of the app.

Reply:
I thought about this. I moved this here to inline with the message that "Firebase Performance Monitoring is successfully initialized!" I am fine to move it back if you think it will eat up the startup time.

Reply:
Discussed offline. Moving the logging to `startWithConfiguration`.